packages/prism/README.md

```diff
@@ -38,5 +38,17 @@ export default ({ children }) => (
 Then, all code blocks in MDX documents wrapped by Layout will be
 syntax highlighted.
 
+Please note that `@theme-ui/prism` uses [`prism-react-renderer`](https://github.com/FormidableLabs/prism-react-renderer), [which does not include all languages supported by Prism](https://github.com/FormidableLabs/prism-react-renderer/blob/master/src/vendor/prism/includeLangs.js) by default. So, if you happen to use one of the missing languages, your code block simply won't show up as highlighted.
+
+However, we have created a workaround. Say that you want to add support for the `R` language. Then you can do that as follows:
```

Comment:
Thanks for this! It'd be good to focus the language on what the user needs to do, rather than making is sound like something *we've* done. Something like this, maybe?

```suggestion
For example, to add support for the `R` language, import the language and pass a custom `Prism` instance to the `CodeBlock` component:
```

Reply:
Love it! Thanks for the suggestion @jxnblk!